docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Now the old exported CSVs from Bittrex can be imported for trades and asset movements.
```

Comment:
There is no reason for a changelog entry. This is already covered by `* :bug:`7119` Removed bittrex as an exchanged added via api key since it has shut down. But users can now import bittrex CSVs in order to get their history into rotki`

```suggestion
```